src/LokalSo/Lokal.cpp

```diff
@@ -143,18 +142,15 @@ std::string Lokal::__curl(std::string path, std::string method,
 	curl_easy_setopt(ch, CURLOPT_HEADERDATA, &data);
 	curl_easy_setopt(ch, CURLOPT_USERAGENT, "Lokal Go - github.com/lokal-so/lokal-go");
 	curl_easy_setopt(ch, CURLOPT_VERBOSE, 0L);
-	curl_easy_setopt(ch, CURLOPT_HTTPHEADER, headers);
+	curl_easy_setopt(ch, CURLOPT_HTTPHEADER, headers.get());
```

Comment:
This header wrapping thing is gruesome. Plus, I don't see the need to
do that. The `slist_append` and `slist_free` here are never touched by
the exception.

It costs extra complexity and an indirect call to the deleter without
code improvement. 

If you really want to wrap it up, create the whole CURL class that
handles the header addition. Like:

```cpp
class Lokal {
private:
	CURLImpl c_;
};


std::string Lokal::curl(/* params */)
{
	c_->setURL(url);
	c_->addHeader("Content-Type: application/json");
	c_->setOpt(/* ... */)
}

```

... and then when the Lokal object is destroyed, the CURLImpl will
also destroyed cleanly withouth the need for `slist_free` and
`slist_append` from the caller (do those internally in CURLImpl).

Something like that is the expected improvement, not just blindly
wrapping everything with unique_ptr. Otherwise, keep it as is in
the C fashioned way.